src/traffic_crashlog/traffic_crashlog.cc

```diff
@@ -262,6 +262,8 @@ main(int /* argc ATS_UNUSED */, const char **argv)
 
   Error("wrote crash log to %s", logname);
 
+  TSfree(logname);
```

Comment:
Wow, this code uses the both `TSfree` and `ats_free`.

Reply:
Yeh, I don't think we should TSfree() here, only ats_free(), right ? TSfree() ought to be only used in plugins.

Reply:
Updated